### PR TITLE
Ethnicity guessed from the author name

### PR DESCRIPTION
**Pull request by glouppe**
Supersedes #70.

### CODE REVIEW

**Comment by glouppe** [comment]
I think it is good to go. Ready for review. @MSusik

**Comment by MSusik** [comment]
3003ef0 should not be included

File: examples/applications/author-disambiguation/ethnicity.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# Copyright (C) 2015 CERN.
+#
+# Beard is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Author disambiguation -- Build an estimator for guessing an author ethnic
+group from his name.
+
+See README.rst for further details.
```
Comment (MSusik) [inline examples/applications/author-disambiguation/ethnicity.py]: No info in README.rst. Please remove the line  or update the file.

File: examples/applications/author-disambiguation/ethnicity.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+"""
+
+import argparse
+import numpy as np
+import pandas as pd
+import pickle
+
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.pipeline import Pipeline
+from sklearn.svm import LinearSVC
```
Comment (MSusik) [inline examples/applications/author-disambiguation/ethnicity.py]: Any reason why particularly this clasifier is used?

**Comment by MSusik** [comment]
After adressing my comments :+1:

File: examples/applications/author-disambiguation/ethnicity.py (hunk 2 above)
Comment (glouppe) [inline examples/applications/author-disambiguation/ethnicity.py]: Linear classifier are often good enough for tf-idf features. I had tried a few alternatives but this one gave decent accuracy without trying too hard.

Comment (MSusik) [inline examples/applications/author-disambiguation/ethnicity.py]: :+1:

**Comment by glouppe** [comment]
Thanks for the review, merging
